postgresql/resource_postgresql_default_privileges_test.go

```diff
@@ -145,3 +145,79 @@ resource "postgresql_default_privileges" "test_ro" {
 		},
 	})
 }
+
+// Test the case where we define default priviliges without specifying a schema. These
+// priviliges should apply to newly created resources for the named role in all schema.
+func TestAccPostgresqlDefaultPrivileges_NoSchema(t *testing.T) {
+	skipIfNotAcc(t)
+
+	// We have to create the database outside of resource.Test
+	// because we need to create a table to assert that grant are correctly applied
+	// and we don't have this resource yet
+	dbSuffix, teardown := setupTestDatabase(t, true, true)
+	defer teardown()
+
+	config := getTestConfig(t)
+	dbName, roleName := getTestDBNames(dbSuffix)
+
+	// Set default privileges to the test role then to public (i.e.: everyone)
+	for _, role := range []string{roleName, "public"} {
+		t.Run(role, func(t *testing.T) {
+
+			// We set PGUSER as owner as he will create the test table
+			var tfConfig = fmt.Sprintf(`
+resource "postgresql_default_privileges" "test_ro" {
+	database    = "%s"
+	owner       = "%s"
+	role        = "%s"
+	object_type = "table"
+	privileges   = %%s
+}
```

Comment:
I would split this out into a separate variable, and define it above the `fmt.Sprintf(` line above, so that it's easier to read, and then you need not have that scary hanging indent at the end either.

Reply:
I have pushed an update to make this test cleaner. Let me know what you think

Reply:
LGTM... I hope the upstream thinks the same, I forgot this was the style the upstream had... missed it when looking at only the diff here :see_no_evil:

Reply:
Yep. I was just following original style but I think it looks much cleaner and easier to read now.